tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Gaufrette\Functional\Adapter;
+
+/**
+ * Class AzureMultiContainerBlobStorageTest
+ * @group AzureBlobStorage
+ * @group AzureMultiContainerBlobStorage
+ */
+class AzureMultiContainerBlobStorageTest extends FunctionalTestCase
```

Comment:
As you're overriding test cases, functional test for `multiContainerMode` disabled are not run. I think it would be better to instantiate two filesystems here and create new test cases for multi container mode.